packages/theme-ui/src/color-modes.js

```diff
@@ -68,4 +68,17 @@ const bodyColor = theme => ({
 
 export const ColorMode = () => <Global styles={bodyColor} />
 
+export const ApplyColorModeFromLocalStorage = () => (
```

Comment:
As far as names go, what about
```suggestion
export const InitializeColorMode = () => (
```